sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -54,7 +58,7 @@ class WorkspaceOperations:
     def __init__(
         self,
         operation_scope: OperationScope,
-        service_client: ServiceClient052022,
+        service_client: ServiceClient102022,
```

Comment:
You'll also need to update the service client object MLClient pass to WorkspaceOperations.

Reply:
Good catch. Done.